src/containers/UserSettings/settings.tsx

```diff
@@ -141,7 +143,7 @@ export const autocompleteOnEnterSetting: SettingProps = {
 export const interfaceVersionInfoField: SettingsInfoFieldProps = {
     title: i18n('settings.about.interfaceVersionInfoField.title'),
     type: 'info',
-    content: process.env.UI_VERSION,
+    content: packageJson.version,
```

Comment:
Have you tried sth like window.ui_version? the same way it is used for react_app_disable_checks

Still feel that using package.json directly can have pitfalls.

Reply:
I tried `index.html`, the problem is that `webpack.DefinePlugin` does not expose vars to html files, so we can use them in js, but not in `index.html` (but all envs passed in command or taken from `.env` file work fine).